apps/publish/content/common.py

```diff
@@ -723,7 +723,7 @@ def _set_updates_for_media_items(self, doc, updates):
             return
 
         for key in DEFAULT_SCHEMA.keys():
-            if doc.get(key) and not updates.get(key):
+            if doc.get(key) and doc.get(key) == updates.get(key):
                 updates[key] = doc[key]
```

Comment:
this seems to me like it would never change the `updates` if it only sets the values which are already equal